website/source/docs/providers/aws/r/lambda_permission.html.markdown

```diff
@@ -27,14 +27,14 @@ resource "aws_lambda_permission" "allow_cloudwatch" {
 resource "aws_lambda_alias" "test_alias" {
   name             = "testalias"
   description      = "a sample description"
-  function_name    = "${aws_lambda_function.test_lambda.arn}"
+  function_name    = "${aws_lambda_function.test_lambda.function_name}"
```

Comment:
The alias accepts either name or the ARN (i.e. both), which is [also documented](https://www.terraform.io/docs/providers/aws/r/lambda_alias.html#function_name), so this is not a real issue, but I agree we can change it and make it look cleaner. 👍